apps/aecore/src/aec_chain.erl

```diff
@@ -253,6 +257,35 @@ get_contract_call(ContractId, CallId, BlockHash) ->
             end
     end.
 
+%%%===================================================================
+%%% Generalized Accounts
+%%%===================================================================
+
+-spec get_ga_call(binary(), aect_call:id(), binary()) ->
+                               {'ok', aega_call:call()} |
+                               {'error', atom()}.
+get_ga_call(Owner, AuthId, BlockHash) ->
+    {value, Account} = get_account(Owner),
```

Comment:
This seems fragile. Account got on top, while call got on block hash.

Though this seems not trivial: even if fetching account at blockhash, who guarantees that account had not been deleted from tx to end of block?

Reply:
Totally right there... but the only way to get hold of the call data is to find it on chain. I guess, the consequence is not that bad if it disappears in the mean time.

Scenario is this:
User posts a GA Tx  or finds one via aepp explorer or so.
Looks up the GA by hash... if something goes wrong, user will try again and in that moment find that it is back in mempool.
Otherwise it is found and data is presented.

Or do I overlook something here?

Reply:
No. This reduces to mostly a (debatable) readability issue, as it makes assumption on lifecycle of items on chain that may be not obvious. I suggested addition of a totally optional comment as a separate PR comment.